bundles/binding/org.openhab.binding.dsmr/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="dsmr"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>DSMR Binding</name>
+    <description>The DSMR Binding obtains data from a Dutch smart meter ("Slimme meter" in Dutch) via the P1-port.</description>
+
+    <config-description>
+        <parameter name="port" type="text" required="true">
+            <label>Serial port</label>
+            <description>The DSMR serial port, e.g. "COM1" for Windows or "/dev/ttyUSB0" for Linux.</description>
+            <default>/dev/ttyUSB0</default>
+        </parameter>
+    </config-description>
+</binding:binding>
```

Comment:
I  don't think this file needs to be in a binding subdirectory.  I'm also wondering how binding.xml and the similarly purposed dsmr.cfg file interact -- which source of configuration data is used, @kaikreuzer?